src/Sql/dbo/Stored Procedures/SsoConfig_ReadByIdentifier.sql

```diff
@@ -4,7 +4,7 @@ AS
 BEGIN
     SET NOCOUNT ON
 
-    SELECT TOP 1
+    SELECT
```

Comment:
So are we supporting multiple ssconfigs per org now? How will that work?

Reply:
I'm not sure if that was ever established so I went for flexibility just in case.  Perhaps they could have multiple configs but only one enabled at a time to allow for experimentation.  /shrug

Reply:
Maybe we should chat about this.